ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransaction.java

```diff
@@ -46,6 +46,12 @@ public abstract class PendingTransaction
   static final int BLOB_SIZE = 131136;
   static final int BLOBS_WITH_COMMITMENTS_SIZE = 40;
   static final int PENDING_TRANSACTION_MEMORY_SIZE = 40;
+
+  static final int DEPOSIT_SIZE = 872;
+  static final int SOURCE_HASH_SIZE = 32;
+  static final int IS_SYSTEM_TX_SIZE = 1;
+  static final int MINT_SIZE = 32;
+
```

Comment:
This is an abstract class, we can move all the optimism and deposit specific stuff into an instance class or an interface with default implementation.  That will help detangle the bits we need to move into a plugin